### DISCUSSION TITLE
Symbolic regression for exchange correlation functionals

### DISCUSSION THREAD

**Discussion by ssahoo41** [MilesCranmer/PySR#547] [Q&A]
Hi Miles, thanks you for creating this software. I was able to learn a lot about symbolic regression by playing around with this package. I have a question. I am currently working on a problem where I have some electron density descriptors using which I want to learn the exchange correlation energy densities. This is a high-dimensional dataset. Is it possible to use PySR to start with an  equation and evolve from there or specify an equation y = f(s) * g(p) where form of f(s) is known?

**Comment by MilesCranmer** [comment]
Hi @ssahoo41,

Good question. There are a few other discussion threads on "seeding" the population if you want to check those out (I don't immediately recall which thread).

One simple thing you could is just transform the target to be `y -> y/f(s)` instead? You could also consider whether a custom loss function makes sense: https://astroautomata.com/PySR/examples/#9-custom-objectives

Cheers,
Miles

**Reply by ssahoo41** [reply]
Thanks for the reply, Miles. Are these discussions about defining a custom operator or could we actually seed the population with some expressions? I have looked at the discussion but I will look again, more carefully. 
Yes I am considering transforming the target variable. A follow-up question I have is that I am working with a high-dimensional data (20 features). Would you recommend directly training a symbolic regression model using PySR for such a dataset? I want to use symbolic regression since I am looking for interpretable models.
